installer/scripts/service_control

```diff
@@ -159,16 +349,22 @@ start_omsagent()
         elif [ -x /bin/systemctl ]; then
             /bin/systemctl start $OMSAGENT_WS
         else
-            echo "Unrecognized service controller to start OMS Agent service" 1>&2
+            echo "ERROR:  Unrecognized service controller to start OMS Agent service" 1>&2
             exit 1
         fi
     fi
+    WS_STATUS=$?
+    if [ $WS_STATUS -eq 0 ]; then
+        wait_until_omsagent_is_running $OMSAGENT_START_WAIT_MAX
+        WS_STATUS=$?
+    fi
+    return $WS_STATUS
 }
 
 stop_omsagent()
 {
-    is_omsagent_running
-    if [ $? -ne 0 ]; then
+    remove_PIDfile_unless_omsagent_running
```

Comment:
Same "react if remove_PIDfile_unless_omsagent_running returns non-zero here" comment here

Reply:
See email exchange on use of remove_PID* procedure.

Reply:
These were discussed around oddments with remove routine.  I think they way it is now is good.